specs/create_Edit_pool.adoc

```diff
@@ -0,0 +1,113 @@
+// vim: tw=79
+
+= Create and Edit Pool
+
+Write a functionality for creating and editing a Pool in Tendrl-Dashboard UI
+
+
+== Problem description
+
+For Creating the new Pool we need to Provide forms with different fields
+* Pool Name
+* Number of Pools to create
+* Type()
+* Replicas
+* Cluster it belongs
+* Journal configration
+* OSD's
+* PG Count
+* Owner
+* Quotas
```

Comment:
@cloudbehl that 128 GB is not quota value, rather its total size available. Thats not a property of pool. It should be coming for the whole pool based on total size available on OSD to data storage.